src/openrct2/title/TitleScreen.cpp

```diff
@@ -217,6 +217,9 @@ extern "C"
         openrct2_write_full_version_info(ch, sizeof(buffer) - (ch - buffer));
         gfx_draw_string(dpi, buffer, COLOUR_BLACK, x + 5, y + 5 - 13);
 
+        // Invalidate screen area
+        gfx_set_dirty_blocks(x, y, x + 500, y + 30);
```

Comment:
`500` is just a large number to cover the width of the string. AppVeyor builds usually have a long version name for example. If there's a simply way to obtain the exact length, please do tell. `30` is simply the line height, to cover both the version and platform labels.